ethcore/wasm/src/tests.rs

```diff
@@ -801,6 +801,7 @@ fn externs() {
 			gas_limit: 0x777777777777u64.into(),
 			last_hashes: Default::default(),
 			gas_used: 0.into(),
+			chain_id: 0.into(), // todo[dvdplm]: check if this is ok
```

Comment:
This should be okay. For all tests we can currently just assume all chain id opcode returns zero, until new test format is defined.